src/tests/unit/services/BavService.test.ts

```diff
@@ -261,7 +261,7 @@ describe("BAV Service", () => {
 		});
 
 		it("Should send event to TxMA without encodedHeader if encodedHeader is empty string", async () => {  
-			await bavService.sendToTXMA("MYQUEUE", "", txmaEventPayload);
+			await bavService.sendToTXMA("MYQUEUE", txmaEventPayload);
```

Comment:
Should this empty string have been removed or moved to be the 3rd parameter passed into this function?

Reply:
Good q - I wanted to test the optional param but not passing it at all (this mirrors what we would actually see when we're integrated too)

Reply:
Thanks for clarifying 😃